sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1047,6 +1047,15 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val CODEGEN_SPLIT_AGGREGATE_FUNC =
+    buildConf("spark.sql.codegen.aggregate.splitAggregateFunc.enabled")
+      .internal()
+      .doc("When true, the code generator would aggregate code into individual methods " +
```

Comment:
It is correct to use aggregate here `... would aggregate code into individual methods ...`? Or we should say `... would split aggregate code into individual methods ...`?

Reply:
oh... right. I'll fix. thansk